style.css

```diff
@@ -76,14 +69,239 @@ h2 {
   color: #7f8cff;
 }
 
+/* Works section */
+.works-container,
+.multi-post-container {
+  display: grid;
+  grid-template-rows: auto;
+  justify-content: center;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #dfe1e6;
+}
+
+.card-one {
+  display: flex;
+  justify-content: center;
+  margin-top: 10px;
+}
+
+.card-two {
+  margin-left: 25px;
+}
+
+.three-elements-row {
+  display: flex;
+  justify-content: space-between;
+  margin-right: 75px;
+  padding: 20px 0;
+}
+
+.three-elements {
+  font-family: "Poppins", arial, helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  background: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  color: #6070ff;
+  letter-spacing: 0.03em;
+}
+
+.tonic-title,
+.multi-post-title,
+.stories-title {
+  font-family: "Poppins", arial, helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172d4d;
+}
```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see you are not using the Correct `font-family`, `font-color` and etc. which makes your design `inconsistent` and looks unprofessional. Please reduce the bottom space and try to make your design as close as possible and have a look at the below-attached image and table.

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-10 17-51-24](https://github.com/Otromascomotu/My-Portfolio-setup-and-mobile-first/assets/66476329/e321b221-d239-4b3e-9d17-82b4914e8fd0)

</td>
<td>

![Screenshot from 2023-06-10 17-51-38](https://github.com/Otromascomotu/My-Portfolio-setup-and-mobile-first/assets/66476329/49f53e8c-e7dd-4997-b462-ebbd359203de)

</td>
</tr>
<tr>
<td>

![Screenshot from 2023-06-10 17-50-45](https://github.com/Otromascomotu/My-Portfolio-setup-and-mobile-first/assets/66476329/bd0be590-8190-45f6-832a-b5d0f1165506)

</td>
<td>

![Screenshot from 2023-06-10 17-50-38](https://github.com/Otromascomotu/My-Portfolio-setup-and-mobile-first/assets/66476329/9ee4f30b-1db3-4ec1-b971-853207d697ec)

</td>
</tr>
</table>

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)